programs/merkle-distributor/src/lib.rs

```diff
@@ -204,12 +203,12 @@ pub struct Claim<'info> {
     #[account(mut)]
     pub to: Account<'info, TokenAccount>,
 
-    /// Who is claiming the tokens.
+    /// CHECK: Who is claiming the tokens.
     #[account(address = to.owner @ ErrorCode::OwnerMismatch)]
-    pub claimant: Signer<'info>,
+    pub claimant: UncheckedAccount<'info>,
```

Comment:
If admin passes their ATA to `to` account, and their wallet address for `claimant` and `payer`, can admin rug it ?

Reply:
Cannot, because the macro `address = to.owner`